docs/index.md

```diff
@@ -90,51 +51,36 @@ The example above uses TypeScript and decorators. You can also easily write LitE
 <div class="responsive-row">
 <div style="flex: 1">
 
-<h2 class="caption">Fast and light</h2>
+<h2 class="caption">Delightfully Declarative</h2>
```

Comment:
Capitalization is inconsistent across these three heads.

Delightfully Declarative
Fast and light
Seamlessly Interoperable

Should either capitalize "Light" or lowercase "declarative" and "interoperable".

We use sentence case for headings in the docs, but the context is so different here I don't think that applies.